cypress/integration/unit_tests/manage_disaster_test.js

```diff
@@ -352,6 +443,66 @@ describe('Unit tests for manage_disaster.js', () => {
                       .get())
         .then((doc) => expect(doc.exists).to.be.false);
   });
+
+  /**
+   * Sets up fresh asset data (no feature collections), and prepares fake page
+   * with damage/map-bounds-related HTML elements.
+   * @return {Cypress.Chainable<Document>}
+   */
+  function setAssetDataAndCreateDamageInputsForScoreValidationTests() {
+    const currentData = createDisasterData(['NY']);
+    currentData['asset_data']['map_bounds_sw'] = '0, 1';
```

Comment:
just setting sw? Asking because above you say "We haven't set anything"

Reply:
rephrased above.